nens/Plot breach data.ipynb

```diff
@@ -0,0 +1,177 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Plot breach data\n",
+    "\n",
+    "Using this script you can plot data regarding breaches in a 3Di model. Plots include breach discharge, velocity, width, depth, waterlevel on both sides of the breach.\n",
+    "\n",
+    "- Author: Emiel Verstegen, 2019\n",
+    "\n",
+    "- email: emiel.verstegen@nelen-schuurmans.nl"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#import libraries, make sure you have them installed on your Python environment\n",
+    "from threedigrid.admin.gridresultadmin import GridH5ResultAdmin\n",
+    "from threedigrid.admin.gridadmin import GridH5Admin\n",
+    "import os\n",
+    "import matplotlib.pyplot as plt\n",
+    "import numpy as np\n",
+    "import math\n",
+    "from pyproj import Proj, transform\n",
+    "from google_drive_downloader import GoogleDriveDownloader as gdd #Needs installing using pip, unlike the others"
```

Comment:
Oei, dat is wel een beetje jammer: dat hier een extra pip-install voor nodig is.
Heeft zo'n google drive document niet ook gewoon een publieke download-url? Of kan je er eentje genereren?
Even binnenhalen met `requests.get()` is wel zo lekker.

Reply:
Nee, niet mogelijk een publieke url te genereren. Zelf ergens hosten dan?